packit/config/common_package_config.py

```diff
@@ -172,6 +172,16 @@ def __init__(
         allowed_committers: Optional[List[str]] = None,
         tmt_plan: Optional[str] = None,
         tf_post_install_script: Optional[str] = None,
+        # # vm-image-build
+        # example: "rhel-86"
+        image_distribution: Optional[str] = None,
+        image_architecture: Optional[str] = None,
+        # example: "aws"
+        image_type: Optional[str] = None,
+        # example: "727920393541"
+        image_account_id: Optional[str] = None,
+        # example: ["foo", "bar-1.2.3"]
+        packages_to_install: Optional[List[str]] = None,
```

Comment:
Are these the additional ones? (=Will the package built in Copr from this checkout be included automatically?)